### ISSUE TITLE
Add call to `set.seed()` in README

### ISSUE THREAD

**Issue by markjrieke** [markjrieke/workboots#12]
for duplication

**Comment by markjrieke** [comment]
Also - add comments to the following:

```r
library(workboots)

# train 100 models on resamples of penguins_train, 
# then make 100 predictions for each penguin in penguins_test
penguins_preds <-
  penguins_wf %>%
  predict_boots(
    n = 100,
    training_data = penguins_train,
    new_data = penguins_test
  )

# summarise the prediction distribution with lower & upper bounds
penguins_preds %>%
  summarise_predictions()

**Comment by markjrieke** [comment]
closed with #20
